### PR TITLE
Update rails 5.2 to rails 6.0

### PR DESCRIPTION
**Pull request by chumaknadya**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ git_source(:github) { |repo_slug| "https://github.com/#{repo_slug}" }
 ruby '~> 2.6'
 
 gem 'ransack', '~> 2.1.1'
-gem 'rails', '~> 5.2.4', '>= 5.2.4.3'
+gem 'rails', '~> 6.0.0'
```
Comment (mod) [inline Gemfile:10]: latest is Rails 6.0.3

Comment (mnaichuk) [reply Gemfile:10]: I will update to 6.0.0 and after to 6.0.3

**Comment by dapi** [comment]
Looks like we need to break this PR on several small PR's and merge them one by one )

Probably extruding rspec updates is a good start.

**Comment by ianeinser** [comment]
Geezzz... that's a huge leapfrog going from major changes all out... it's wiser to do modules by modules.. if that ever possible in ruby rails

**Pull request closed without merging**